release-notes/unreleased/IGAPP-263-open-news-item.yml

```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-263
+show_in_stores: true
+platforms:
+  - android
+  - ios
+en: Clicking on push notifications now opens the correct news.
+de: Beim Anklicken von Push Notifications wird nun die richtige Nachricht geöffnet.
```

Comment:
(minor/optional) both english and german sound a little like before this updated, the wrong news was opened.